gateway/src/main.rs

```diff
@@ -49,6 +50,10 @@ struct Args {
     #[clap(default_value_t = LogFormat::default())]
     log_format: LogFormat,
 
+    /// Run database migrations manually then exit.
+    #[arg(long)]
+    run_migrations: bool,
```

Comment:
Could we call this something like `--run-migrations-only`, to make it clear that we still run migrations when it's not set?

Reply:
yep